### ISSUE TITLE
no page found for every endpoint

### ISSUE THREAD

**Issue by papi028** [BeamlakAschalew/flixquest-api#29]
getting 
{
    "message": "",
    "error": "page not found"
}

for every endpoint i use like this example
https://flixquest-api.vercel.app/flixhq/watch-movie?tmdbId=18785

**Comment by BeamlakAschalew** [comment]
> getting 
> {
>     "message": "",
>     "error": "page not found"
> }
> 
> for every endpoint i use like this example
> https://flixquest-api.vercel.app/flixhq/watch-movie?tmdbId=18785

This happened because I had to remove all the routes associated with movie-web extension because it is removed from npm, this route wasn't providing streaming links right?

**Comment by Tommy0412** [comment]
why? Simply install then from github ?

**Comment by papi028** [comment]
> > getting
> > {
> > "message": "",
> > "error": "page not found"
> > }
> > for every endpoint i use like this example
> > https://flixquest-api.vercel.app/flixhq/watch-movie?tmdbId=18785
> 
> This happened because I had to remove all the routes associated with movie-web extension because it is removed from npm, this route wasn't providing streaming links right?

yea just this error that comes nothing else

**Comment by BeamlakAschalew** [comment]
> why? Simply install then from github ?

Yes they exist but most of the endpoints like flixhq, zoe etc won't work anyways, I'll integrate the new ones when I get time

**Comment by papi028** [comment]
any updates?

**Comment by BeamlakAschalew** [comment]
> any updates?

The movie web extension was deprecated and removed from npm and it was resulting in an error, that's why it's removed.

**Comment by papi028** [comment]
yea i seen that also , so thats its?do we say this one dead too or you
 tryna find something

**Comment by BeamlakAschalew** [comment]
> yea i seen that also , so thats its?do we say this one dead too or you
>  tryna find something

Try out consumet, @consumet but only anime works now i think

**Comment by papi028** [comment]
i already use consumet i was using their anime providers and the flixhq provider buh when flixhq went down that why i was looking for alternatives
